modules/administration-guide/pages/configuring-editors-download-urls.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: PROCEDURE
+:description: Configuring editors download urls
+:keywords: administration guide, dashboard, editors
+:navtitle: Configuring editors download urls
+
+[id="configuring-editors-download-urls"]
+= Configuring editors download urls
+
+Learn how to configure editors download urls. This feature is valuable in air-gapped environments where editors cannot be retrieved from the public internet.
```

Comment:
Same concern as in https://github.com/eclipse-che/che-operator/pull/2026/files#r2269472787 
we must clarify which exact editors could benifit from this property